experiment-runner/ExperimentOrchestrator/Experiment/Run/RunController.py

```diff
@@ -6,8 +8,45 @@
 from ProgressManager.Output.OutputProcedure import OutputProcedure as output
 
 class RunController(IRunController):
+    # Start EnergiBridge measurements
+    def start_eb(self):
+        if not self.config.self_measure:
+            return
+
+        eb_args = [self.config.self_measure_bin, "--summary",
+                   "-o", "/dev/null", "--", "sleep", "1000000"]
+
+        try:
+            self.eb_proc = subprocess.Popen(eb_args, stdout=subprocess.PIPE,
```

Comment:
Whenever I run `energibridge`, I also use a file to output the logs, just in case I miss it at runtime. Might be a good idea to have it here too. We can see what @iivanoo thinks.

Reply:
I agree. It can be "promoted" to a configuration parameter of the plugin, so that it will be the user deciding whether they want to persist the logs on a file or not.

Reply:
Should I fix this and add another PR?

Reply:
I think you can fixing it in the same PR is perfectly fine